heureka/ui/src/helpers.js

```diff
@@ -20,6 +20,14 @@ export const parseError = (error) => {
   return errMsg
 }
 
+export const formatLabel = (label) => {
```

Comment:
It looks like this does a very similar thing to the `humanizeString` function you already have in utils. Is this helper really necessary?

Reply:
there is a slight difference that in formatLabel, all vocabularies are started with capital case. But it is not that important, I deleted this function and used the humanize string to keep the consistency in greenhouse as the Alerts filter labels are using this formatter as well.